lib/dfe/reference_data/itt/incentives.rb

```diff
@@ -30,6 +30,126 @@ module ITT
       INCENTIVES = DfE::ReferenceData::HardcodedReferenceList.new(
         # SEE https://www.gov.uk/government/publications/initial-teacher-training-itt-bursary-funding-manual
         {
+          ## 2025-2026
+          'bf4ea64c-7f0d-4604-855d-b2d7a8fa3db8' => {
+            subject_name: 'mathematics',
+            itt_categories_id: '71cbc6f2-9b24-4504-86f5-979e573de058',
+            academic_cycle: ['2026'],
+            bursary_amount: '30000',
+            scholarship: '32000',
+            notes: 'physics, mathematics, computing, chemistry 2025-2026'
+          },
+          'f0157f2e-6953-4e4b-b39b-575e34f1f975' => {
+            subject_name: 'physics',
+            itt_categories_id: 'd059a9cd-9b70-49bf-8194-9a71081e6cb9',
+            academic_cycle: ['2026'],
+            bursary_amount: '30000',
+            scholarship: '32000',
+            notes: 'physics, mathematics, computing, chemistry 2025-2026'
+          },
+          '0d497364-a854-40f5-be0d-bf490ab8cb1b' => {
+            subject_name: 'chemistry',
+            itt_categories_id: '167b889a-4cbb-47ff-a196-4103cb7ea43d',
+            academic_cycle: ['2026'],
+            bursary_amount: '28000',
+            scholarship: '30000',
+            notes: 'physics, mathematics, computing, chemistry 2025-2026'
+          },
+          '39134b18-dfd0-4ff5-b4d4-a50af6fd9e40' => {
+            subject_name: 'computing',
+            itt_categories_id: 'bc0a97ed-c09c-4688-9dce-b4ea888b6c51',
+            academic_cycle: ['2026'],
+            bursary_amount: '28000',
+            scholarship: '30000',
```

Comment:
update to
bursary_amount: '29000',
scholarship: '31000',

Reply:
Updated to the new value